code/modules/shuttle/on_move.dm

```diff
@@ -297,24 +297,29 @@ All ShuttleMove procs go here
 
 /************************************Item move procs************************************/
 
-/obj/item/storage/pod/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
+/obj/item/storage/pod/afterShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
+	. = ..()
 	unlocked = TRUE
 	// If the pod was launched, the storage will always open.
-	return ..()
 
 /************************************Mob move procs************************************/
 
 /mob/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
 	if(!move_on_shuttle)
-		return 0
-	. = ..()
-	if(!.)
 		return
-	if(client)
+	. = ..()
+
+/mob/afterShuttleMove(list/movement_force, shuttle_dir, shuttle_preferred_direction, move_dir)
+	. = ..()
+	if(client && movement_force)
+		var/shake_force = max(movement_force["THROW"], movement_force["KNOCKDOWN"])
 		if(buckled)
-			shake_camera(src, 2, 1) // turn it down a bit come on
-		else
-			shake_camera(src, 7, 1)
+			shake_force *= 0.25
+		shake_camera(src, shake_force, 1)
+
+/mob/living/beforeShuttleMove(turf/newT, rotation, move_mode)
+	. = ..()
+	Stun(1) //Just making sure people aren't moving around between beforeShuttleMove ticks
```

Comment:
this has sideffects; everyone will drop held items, even if buckled, and it doesn't work on stun-immune things.

Reply:
This is pointless anyway I realized, forgot to remove this after testing